### ISSUE TITLE
Wrong output if exports use the same name

### ISSUE THREAD

**Issue by peterjuras** [yamafaktory/shrimpit#37]
Thanks again for creating this tool.

Currently all export and import names are flattened when analyzing unused exports. This leads to issues when multiple files have exports with the same name. Example:

```js
// a.js
export function a() {

}
```

```js
// b.js
export function a() {

}

export function b() {

}
```

```js
// c.js
import { a } from './a';
import { b } from './b';

a();
b();
```

shrimpit! output:
```
 Shrimpit!

 > Files tree

{ '.': null,
  'a.js': { exports: [ 'a' ], imports: [] },
  'b.js': { exports: [ 'a', 'b' ], imports: [] },
  'c.js': { exports: [], imports: [ 'a', 'b' ] } }

 > Unused exports

All Clear Ahead, Captain.
```

This is wrong, because export **a** from file **b.js** is not used anywhere. However it gets declared as used, since c.js is using the named export a from a.js.

In general I think it might more sense to always use the full path identifier for imports and exports, since it makes it more readable when fixing unused exports. As an example, I'm currently cleaning up a codebase with a lot of react components and in conjunction with bug #36 my output is often something like this:

```
 Shrimpit!

 > Unused exports

[ 'constructor',
  'componentDidMount',
  'componentWillReceiveProps',
  'setState',
  'error',
  'constructor',
  'componentDidMount',
...
```

Apart from the real issue mentioned above, not using the full path identifier makes it hard to find the file that has the unused export.

**Comment by yamafaktory** [comment]
Hi @peterjuras,

Thanks for the report! It's been a long time since I've touched this project... I've just started to do some refactoring and will try to address this issue & #36 too soon.

**Comment by yamafaktory** [comment]
@peterjuras the latest version should fix both issues, please give it a try and keep me updated! Thanks!

**Comment by peterjuras** [comment]
Hi @yamafaktory thanks for the very quick fix!

I can confirm that this issue here is fixed, however #36 still has some troubles, I will comment there.

Thanks a lot!
